modules/machine-config-node-disruption.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="machine-config-node-disruption_{context}"]
+= Understanding node restart behaviors after machine config changes
+
+By default, when you make certain changes to the fields in a `MachineConfig` object, the Machine Config Operator (MCO) drains and reboots the nodes associated with that machine config. However, you can create a _node disruption policy_ in the MCO namespace that defines a set of changes to some Ignition config objects that would require little or no disruption to your workloads.
+
+A node disruption policy allows you to define the configuration changes that cause a disruption to your cluster, and which changes do not. This allows you to have less downtime when making small machine configuration changes in your cluster.
+
+Note that there are machine config changes that do not require a reboot, regardless of any node disruption policies. For more information, see _About the Machine Config Operator_.
+
+For example, you could configure a policy so that sudo reconfigurations do not require a node drain and reboot. Or, you could configure your cluster so that updates to `sshd` are applied with only a reload of that one service.
+
+////
+The following is a planned feature
+You could also allow the removal of mirroring rules from an ICSP, ITMS, IDMS object without workload disruption.
```

Comment:
Assuming these are changes to `/etc/containers/registries.conf`, the feature in its current state already allows for this.

Reply:
@djoshy @sergiordlr Was the registries.conf changes part of the feature QE tested? I didn't see it in the [Polarion test cases](https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitems/testcase?query=trello%3AMCO%5C-507).
I would love an example node disruption policy for registries.conf changes, if it was.

Reply:
I don't think there is a specific test for it. You can treat it like a regular file, and give it a None action, making any changes to it disruption less. You'll see by default it has a Special action which is the "old" policy for changes to the /etc/containers/registries.conf file